example/ios/Podfile

```diff
@@ -1,5 +1,5 @@
 # Uncomment this line to define a global platform for your project
-platform :ios, '9.3'
+platform :ios, '11.0'
```

Comment:
The example project was using the LTS release. As per the [flutter_ffmpeg](https://pub.dev/packages/flutter_ffmpeg) package:

Main releases:
```
platform :ios, '11.0'
```
LTS releases:
```
platform :ios, '9.3'
```

So, the ios deployment target `9.3` should work fine.